plugins/woocommerce/client/admin/client/settings-payments/components/country-selector/utils.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Internal dependencies
+ */
+import { Item } from './types';
+
+// Truncate the option label if it exceeds the max length.
+export const truncateOptionlabel = ( label: string, maxLength: number ) => {
```

Comment:
I think it might be cleaner to use css rule `text-overflow: ellipsis` for this

Reply:
Good suggestion! I’ve replaced the JS method with `text-overflow: ellipsis`.